src/proto/Crypto_org.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+syntax = "proto3";
+
+package TW.Crypto_org.Proto;
```

Comment:
can you clean up all the "TODO" and resue Cosmos.proto?

Reply:
Crypto.org blockchain is based on Cosmos.
in coin definition, we define `"blockchain" : "Cosmos"`
Does the proto files are needed ?

If I look at Thorchain, I don't find any proto file

Reply:
Right, since CyptoOrg reuses Cosmos Entry and Signer implementation, the Cosmos SigningInput and SigningOutput types are reused.  No need fot CryptoOrg proto file.
Please remove the .proto file.